ui.tests/test-module/specs/ruleeditor/customFunction.runtime.spec.js

```diff
@@ -63,8 +63,9 @@ describe('Form with custom functions configured in client lib', () => {
         cy.get(`#${textbox1}`).find("input").should('have.value', "test")
```

Comment:
Can you also write e2e test case via custom function for the new submitForm API which optionally takes validateForm, this would act as a reference

Reply:
Added: https://github.com/adobe/aem-core-forms-components/pull/1191/files#diff-507c43065e09893cfa5cacac09182c6fe9e18439b7cc79f21eb0c3a27f79b892R244